PrismSampleApp/PrismSampleApp/Views/LoginPage.xaml

```diff
@@ -29,7 +29,7 @@
                                     <Frame BackgroundColor="SkyBlue" HeightRequest="40" WidthRequest="40" CornerRadius="30" Padding="0" Margin="5">
                                         <Image Source="https://www.findool.com/images/avatars/avatar.png" Aspect="AspectFill" Margin="0"/>
                                     </Frame>
-                                    <Entry Text="{Binding Username}" Placeholder="Email" TextColor="HotPink" FontAttributes="None" HorizontalOptions="FillAndExpand" Margin="0,0,20,0"/>
+                                    <Entry Text="{Binding Username}" Placeholder="UserName" TextColor="HotPink" FontAttributes="None" HorizontalOptions="FillAndExpand" Margin="0,0,20,0"/>
```

Comment:
Localize all the Text in xaml. placeholder text as well

Reply:
Localized all the texts.